site/app/controllers/admin/UsersController.php

```diff
@@ -398,6 +398,30 @@ public function deleteUser(string $type = 'users'): RedirectResponse {
         return new RedirectResponse($this->core->buildCourseUrl([$type]));
     }
 
+    /**
+     * @param string $type
+     * @Route("/courses/{_semester}/{_course}/demote_grader", methods={"POST"})
+     * @return RedirectResponse
+     */
+    public function demoteGrader(string $type = 'users'): RedirectResponse {
```

Comment:
Since this is the demote_graders route, couldn't we just assume that the type will always be a grader and thus redirect always to the graders page? And maybe the same should happen for the deleteUser function to only focus on users (students) and not support graders.